swaglyrics/cli.py

```diff
@@ -96,12 +96,14 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 	:param make_issue: whether to make an issue on GitHub if song unsupported
 	:return: lyrics if song playing
 	"""
+	"""
```

Comment:
why is this necessary?

Reply:
Because After commenting out the code, Code does required functioning for the unsupported song also, after the addition of 'From' inside the regex expression.